### PR TITLE
Store registration

### PR DESCRIPTION
**Pull request by mrafaeli**

### CODE REVIEW

File: app/controllers/stores/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Stores::SessionsController < Devise::SessionsController
+  # before_action :configure_sign_in_params, only: [:create]
+  def after_sign_in_path_for(resource)
```
Comment (monkeyci) [inline app/controllers/stores/sessions_controller.rb]: Unused method argument - resource. If it's necessary, use _ or _resource as an argument name to indicate that it won't be used. You can also write as after_sign_in_path_for(*) if you want the method to accept any arguments but don't care about them.

File: app/controllers/stores/registrations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class Stores::RegistrationsController < Devise::RegistrationsController
+  # before_action :configure_sign_up_params, only: [:create]
+  # before_action :configure_account_update_params, only: [:update]
+  def after_sign_in_path_for(resource)
```
Comment (monkeyci) [inline app/controllers/stores/registrations_controller.rb]: Unused method argument - resource. If it's necessary, use _ or _resource as an argument name to indicate that it won't be used. You can also write as after_sign_in_path_for(*) if you want the method to accept any arguments but don't care about them.

File: app/assets/stylesheets/user_landing.scss [hunk 3]
```diff
@@ -29,7 +29,8 @@
 .prompt {
   text-align: center;
 
-  &__btn, input[type="submit"] {
+  &__btn,
+  input[type="submit"] {
```
Comment (monkeyci) [inline app/assets/stylesheets/user_landing.scss]: Unexpected qualifying type selector                   selector-no-qualifying-type<br>Expected single quotes                                string-quotes

File: app/assets/stylesheets/user_landing.scss [hunk 4]
```diff
@@ -41,7 +42,7 @@
   }
 
   &__input,
-  input[type="text"] {
+  input[type='text'] {
```
Comment (monkeyci) [inline app/assets/stylesheets/user_landing.scss]: Unexpected qualifying type selector                   selector-no-qualifying-type

File: app/views/stores/sessions/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,30 @@
+
+
+
```
Comment (imarquezc) [inline app/views/stores/sessions/new.html.erb]: de mañoso igual, pero estos espacios en blanco deberías sacarlos

**Pull request merged**